SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -352,6 +352,11 @@ FLocalDeploymentManager::ERuntimeStartResponse FLocalDeploymentManager::StartLoc
 	EAutomationControllerModuleState::Type TestState = AutomationController->GetTestState();
 	bTestRunnning = TestState == EAutomationControllerModuleState::Type::Running;
 
+	if (CallBack)
+	{
+		CallBack(/*bSuccess =*/true);
```

Comment:
Omega Nit: I would suggest using a similar comment format as you did for the call to StartInspectorProcess for consistency. One other tiny detail is that this comment format is usually used for default arguments so is very subtly confusing on first glance.

Reply:
good points, changed